storage/backup/backup-cli/proptest-regressions/storage/command_adapter/tests.txt

```diff
@@ -0,0 +1 @@
+cc 2680b092e9222788669cf48b288accf336d8d0b12c60fb312bcd12c5f98ddd99
```

Comment:
Are you sure you want to check this in?   Chances are the code will change enough over time this wont even trigger the dup issue you saw.

Reply:
K. Realized it's not a common practice in our code base to check that in.